app/views/demo_mailer/send_worksheets_to_teacher.html.erb

```diff
@@ -16,14 +17,18 @@
               <ul>
                 <p>Students:</p>
                 <% work_group[:users].each do |student| %>
-                  <li><%= student['name'] %></li>
+                   <div class="row">
+                      <div class="col-sm-2"> <span><div class="circle" style='background-color: <%= student['pen_colour'] %>'></div></span>
+                        <p><%=student['name'] %></p>  
+                      </div>
+                    </div>
```

Comment:
This puts a little colored dot next to the student's name. I don't know if this is the best way to do it but it was one solution that I found. Also, I only tested this on the web browser so not quite sure how it'll look in email.

Reply:
let's add the colour name too just in case